personal_graph/clients.py

```diff
@@ -58,18 +60,32 @@ def _create_default_client(self):
 
 
 @dataclass
-class OpenAILLMClient(APIClient):
+class LiteLLMClient(APIClient):
     base_url: str = ""
     model_name: str = "openai/gpt-3.5-turbo"
 
-    def __post_init__(self):
-        self.client = self._create_default_client()
+    def __post_init__(self, *args, **kwargs):
+        self.client = self._create_default_client(*args, **kwargs)
 
-    def _create_default_client(self):
+    def _create_default_client(self, *args, **kwargs):
         return openai.OpenAI(
             api_key="",
             base_url=os.getenv("LITE_LLM_BASE_URL", self.base_url),
             default_headers={
                 "Authorization": f"Bearer {os.getenv('LITE_LLM_TOKEN', '')}"
             },
+            *args,
+            **kwargs,
         )
+
+
+@dataclass
+class OpenAIClient(APIClient):
+    api_key: str = os.getenv("OPENAI_API_KEY", "")
+    model_name: str = "gpt-3.5-turbo"
```

Comment:
doesnt seem to be required or used

Reply:
Yes, we are using it inside graph generator because previously we were taking function_calling_model param for OpenAITextToGraphParser class , so instead of that we are taking the model name from clients itself.